tests/Support/TestingShipment.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+namespace Tests\Support;
+
+use Closure;
+use Exception;
+use GuzzleHttp\Handler\MockHandler;
+use GuzzleHttp\HandlerStack;
+use GuzzleHttp\Middleware;
+use GuzzleHttp\Psr7\Response;
+use MelhorEnvio\MelhorEnvioSdkPhp\Shipment;
+
+class TestingShipment extends Shipment
+{
+    private HandlerStack $handlerStack;
```

Comment:
Fica bem mais verboso, mas acho que fica melhor pra entender, o que acha ?

```suggestion
    private HandlerStack $httpClientMiddlewareStack;
```

Reply:
O que acha então de importar o HandlerStack como GuzzleHandlerStack ? Ficaria mais explicito,  pois pelo menos da primeira vez que li as alterações demorei a entender o que exatamente era um HandlerStack ou "stack", foi só depois que entendi que se tratava de um stack de middleware que passou a fazer sentido pra mim